bundle/manifests/kogito-operator.clusterserviceversion.yaml

```diff
@@ -541,4 +541,4 @@ spec:
   provider:
     name: Red Hat
   replaces: kogito-operator.v1.7.0
```

Comment:
Is the `replaces` adjusted in some separate place?

Reply:
it should have been here ...

Reply:
```suggestion
  replaces: kogito-operator.v1.8.0
```